ansible/group_vars/kafka.yml

```diff
@@ -0,0 +1,4 @@
+datadog_integrations:
+  - kafka
+  - kafka_consumer
+  - zk
```

Comment:
this should actually be in ` ansible/group_vars/zookeeper.yml` but i think since zookeeper and kafka are on the same machine the datadog_integrations variable gets overridden so it doesn't properly deploy the zk.yaml file